test/ReactViews/Custom/Chart/MomentPointsChartSpec.tsx

```diff
@@ -18,11 +24,15 @@ describe("MomentPointsChart", function () {
       { x: new Date("2020-05-29"), y: 0.5 },
       { x: new Date("2020-05-30"), y: 0.5 }
     ],
+    domain: { x: [new Date("2020-05-25"), new Date("2020-05-30")], y: [0, 10] },
```

Comment:
I assume the domain relates to the points. I don't know if domain can contain dates, but nothing complains when I put them there, so I did.

I took the other values from one of the chartItems in BottomDockChartSpec.tsx.